elasticdl/python/common/args.py

```diff
@@ -463,6 +463,13 @@ def add_common_args_between_master_and_worker(parser):
         "according to the distributed strategy, "
         'e.g. "ParameterServerStrategy"',
     )
+    parser.add_argument(
+        "--record_failure_tolerance",
```

Comment:
If this is a percentage, then maybe rename this to "failure_tolerance_percentage"?

Reply:
Fixed